assets/scripts/aircraft.js

```diff
@@ -714,16 +753,6 @@ var Aircraft=Fiber.extend(function() {
       else {
         this.hideStrip();
 
-        // Fly away!
-        this.fms.setCurrent({
-          navmode: "heading",
-          heading: this.radial,
-          turn: null,
-          hold: false,
-          altitude: 20000,
-          speed: 330,
-        });
-
```

Comment:
The intent of this code was to ensure an aircraft outside of control would always leave the game.  For example an aircraft can be in fix track mode but end up circling the fix indefinitely without a guard like this.